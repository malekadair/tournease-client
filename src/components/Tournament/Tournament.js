import React, { Component } from "react";
import { Link } from "react-router-dom";

class Tournament extends Component {
  render() {
    const { name, game, details, id } = this.props.tourney;
    console.log("tournament props", this.props);
    return (
      <div>
        <h1>{name}</h1>
        <h2>{game}</h2>
        <p>{details}</p>
        <Link to={`/${id}`}>More details</Link>
      </div>
    );
  }
}

export default Tournament;
