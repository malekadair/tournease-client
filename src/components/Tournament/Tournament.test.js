import React from "react";
import toJson from "enzyme-to-json";
import Tournament from "./Tournament";

describe(`Tournament component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a .Tournament by default", () => {
    const wrapper = <Tournament />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Tournament given props", () => {
    const wrapper = <Tournament {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
