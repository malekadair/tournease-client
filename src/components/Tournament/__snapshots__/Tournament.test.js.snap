// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tournament component renders a .Tournament by default 1`] = `undefined`;

exports[`Tournament component renders the Tournament given props 1`] = `undefined`;
